front/app/containers/Admin/projects/project/poll/WrongMaxChoiceIndicator.test.tsx

```diff
@@ -62,47 +61,6 @@ describe('<WrongMaxChoiceIndicator/>', () => {
       expect(Wrapper).toMatchSnapshot();
     });
   });
-  describe('renders the error message when max answers is not as expected', () => {
```

Comment:
These are the only ones I deleted as it wasn't worth spending that much time on them.